src/main.rs

```diff
@@ -155,6 +156,10 @@ pub struct Args {
     /// Path to the nvim binary
     pub nvim_bin_path: Option<String>,
 
+    #[arg(long)]
+    /// TCP address to nvim server
+    pub server: Option<SocketAddr>,
```

Comment:
call this server like nvim even though we don't support unix pipes (yet) or `tcp`?

Reply:
So long as the intent is to allow for passing Unix FDs eventually through the same command, that seems fine to me. But we might want to rephrase the comment to something like: `Nvim server to connect to (currently TCP only)`

Reply:
used ` Nvim server to connect to (currently TCP only)`